nanovg.cabal

```diff
@@ -33,6 +38,7 @@ library
                        NanoVG.Internal.FFIHelpers
                        NanoVG.Internal.FixedVector
                        NanoVG.Internal.GL3
+                       NanoVG.Internal.GLES3
```

Comment:
Should this be guarded using the `gles3` flag? It looks like it won’t be particularly useful if it is not enabled.
Similarly it would the `GL3` module should probably not be enabled when `gles3` is enabled.

Reply:
Yes, you could actually guard it using the flag, but disabling GL3 would lead to problems, because GLES3 depends on the CreataeFlags from GL3.